doc/source/analysis.rst

```diff
@@ -1,5 +1,400 @@
+
+>>> gu = ig.union([g, g2, g3])
+
+Similarly for the intersection (nodes/edges present in all are kept):
+
+>>> gu = ig.intersection([g, g2, g3])
+
+These two operations preserve attributes and can be performed with a few variations. The most important one is that vertices can be matched across the graphs by id (number) or by name.
+
+These and other operations are also available as methods of the :class:`Graph` class:
+
+>>> g.union(g2)
+>>> g.intersection(g2)
+>>> g.disjoint_union(g2)
+>>> g.difference(g2)
+>>> g.complementer()  # complement graph, same nodes but missing edges
+
+and even as numerical operators:
+
+>>> g |= g2
+>>> g_intersection = g and g2
+
+Topological sorting
++++++++++++++++++++
+To sort a graph topologically, use :meth:`Graph.topological_sorting`:
+
+>>> g = ig.Graph.Tree(10, 2, mode=ig.TREE_OUT)
+>>> g.topological_sorting()
+
+Graph traversing
++++++++++++++++++++++
+A common operation is traversing the graph. |igraph| currently exposes breath-first search (BFS) via :meth:`Graph.bfs` and :meth:`Graph.bfsiter`:
+
+>>> [vertices, layers, parents] = g.bfs()
+>>> it = g.bfsiter()  # Lazy version
+
+A depth-first search function is in the works.
+
+To perform a random walk from a certain vertex, use :meth:`Graph.random_walk`:
+
+>>> vids = g.random_walk(0, 3)
+
+Pathfinding and cuts
+++++++++++++++++++++
+Several pathfinding algorithms are available:
+
+- :meth:`Graph.shortest_paths` or :meth:`Graph.get_shortest_paths`
+- :meth:`Graph.get_all_shortest_paths`
+- :meth:`Graph.get_all_simple_paths`
+- :meth:`Graph.spanning_tree` finds a minimum spanning tree
+
+As well as functions related to cuts and paths:
+
+- :meth:`Graph.mincut` calculates the minimum cut between the source and target vertices
+- :meth:`Graph.st_mincut` - as previous one, but returns a simpler data structure
+- :meth:`Graph.mincut_value` - as previous one, but returns only the value
+- :meth:`Graph.all_st_cuts`
+- :meth:`Graph.all_st_mincuts`
+
+Global properties
++++++++++++++++++++++
+A number of global graph measures are available.
+
+Basic:
+
+- :meth:`Graph.diameter` or :meth:`Graph.get_diameter`
+- :meth:`Graph.girth`
+- :meth:`Graph.radius`
+- :meth:`Graph.average_path_length`
+
+Distributions:
+
+- :meth:`Graph.degree_distribution`
+- :meth:`Graph.path_length_hist`
+
+Connectedness:
+
+- :meth:`Graph.all_minimal_st_separators`
+- :meth:`Graph.minimum_size_separators`
+- :meth:`Graph.feedback_arc_set`
+- :meth:`Graph.cut_vertices` or :meth:`Graph.articulation_points`
+
+Cliques and motifs:
+
+- :meth:`Graph.clique_number` (aka :meth:`Graph.omega`)
+- :meth:`Graph.cliques`
+- :meth:`Graph.maximal_cliques`
+- :meth:`Graph.largest_cliques`
+- :meth:`Graph.motifs_randesu` and :meth:`Graph.motifs_randesu_estimate`
+- :meth:`Graph.g.motifs_randesu_no` counts the number of motifs
+
+Structural:
+
+- :meth:`Graph.edge_connectivity` or :meth:`Graph.edge_disjoint_paths` or :meth:`Graph.adhesion`
```

Comment:
This is basically unweighted `mincut_value`. Since "Structural" is so vague, maybe remove this section, and mention these where cuts are discussed? Vertex and edge connectivity are merely the size of the minimum edge and vertex cuts.

Reply:
yep done!